src/test/java/org/jenkinsci/plugins/envinject/BuildCauseRetrieverTest.java

```diff
@@ -1,121 +0,0 @@
-package org.jenkinsci.plugins.envinject;
```

Comment:
Seems Git has not noticed the move operation. It will cause conflicts with other PRs, so it's better to revert the change